packages/core-data/src/actions.js

```diff
@@ -798,6 +798,59 @@ export const saveEditedEntityRecord =
 		return await dispatch.saveEntityRecord( kind, name, record, options );
 	};
 
+/**
+ * Action triggered to update only specified properties for the entity.
+ *
+ * @param {string} kind        Kind of the entity.
+ * @param {string} name        Name of the entity.
+ * @param {Object} recordId    ID of the record.
+ * @param {Array}  itemsToSave List of entity properties or property paths to save.
+ * @param {Object} options     Saving options.
+ */
+export const __experimentalUpdateSpecifiedEntityEdits =
+	( kind, name, recordId, itemsToSave, options ) =>
+	async ( { select, dispatch } ) => {
+		if ( ! select.hasEditsForEntityRecord( kind, name, recordId ) ) {
+			return;
+		}
+		const edits = select.getEntityRecordNonTransientEdits(
+			kind,
+			name,
+			recordId
+		);
+
+		// Get the base record from the datase to apply the edits to.
+		const base = select.getEntityRecord( kind, name, recordId ) || {};
+
+		// Use the base record as the starting point for the edits.
+		const editsToSave = base;
```

Comment:
```suggestion
		// Use the base record as the starting point for the edits.
		const editsToSave = select.getEntityRecord( kind, name, recordId ) || {};
```

Reply:
I thought about that, too, I decided to create a new one to be explicit about the difference with `__experimentalSaveSpecifiedEntityEdits` and avoid changing the existing functionality that used by other parts of the codebase too.

Reply:
Still, if we find that the behavior of `__experimentalUpdateSpecifiedEntityEdits` is the originally intended behavior of `__experimentalSaveSpecifiedEntityEdits,` we can add the changes from this PR to `__experimentalSaveSpecifiedEntityEdits` instead of creating the new function.